llvm/tools/llvm-cas-dump/CASDWARFObject.cpp

```diff
@@ -62,6 +64,47 @@ struct MachOHeaderParser {
 };
 } // namespace
 
+Error CASDWARFObject::discoverDebugInfoSection(ObjectRef CASObj,
```

Comment:
Maybe you did this already in the header, but if not: this code could use some comments

Reply:
Hmm, there is no documentation for `discoverDwarfSections` I will add that too